packages/tailwindcss/src/plugin-api.ts

```diff
@@ -186,27 +212,400 @@ export function buildPluginApi(designSystem: DesignSystem, ast: AstNode[]): Plug
         })
       }
     },
+    theme(path) {
+      let cssValue = readFromCss(designSystem.theme, path)
+
+      if (typeof cssValue !== 'object') {
```

Comment:
Should we check for `null` here?

Reply:
Technically it's not necessary because `typeof null === "object"` _and_ we want that case to fall through. I've added an explicit check anyway + a clarifying comment just so the intent of the code is clearer.